colossalai/nn/layer/vanilla/layers.py

```diff
@@ -254,3 +254,37 @@ def reset_parameters(self, weight_initializer, bias_initializer):
 
     def forward(self, input_: Tensor) -> Tensor:
         return F.linear(input_, self.weight, self.bias)
+
+
+@LAYERS.register_module
+class VanillaLayerNorm(nn.Module):

```

Comment:
Is it different from `torch.nn.LayerNorm`?

Reply:
@ver217 A little. We allow it to ignore bias.